config.py

```diff
@@ -12,18 +12,15 @@ def init_app(app):
 
 class DevelopmentConfig(Config):
     DEBUG = True
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DEV_DATABASE_URL') or \
-    'sqlite:///' + os.path.join(basedir, 'dev-db.sqlite')
+    SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/mrm_db'
```

Comment:
I think we should not hard code DATABASE_URI variables. It's better to  use environment variables such as SQLALCHEMY_DATABASE_URI = os.getenv(DEV_DATABASE_URL).

Reply:
okay, Let me fix that